client/lib/i18n-utils/switch-locale.js

```diff
@@ -79,9 +80,10 @@ export function switchCSS( elementId, cssUrl, callback = noop ) {
 /**
  * Loads a css stylesheet into the page
  * @param {string} cssUrl - a url to a css resource to be inserted into the page
+ * @param {Element} currentLink - a <link> DOM element that we want to use as a reference for stylesheet order
  * @param {Function} callback - a callback function to be called when the CSS has been loaded (after 500ms have passed).
  */
-function loadCSS( cssUrl, callback = noop ) {
+function loadCSS( cssUrl, currentLink = null, callback = noop ) {
```

Comment:
Why do we need to define a default value for `currentLink`?

Reply:
no need, you are right